Marlin/SanityCheck.h

```diff
@@ -895,27 +891,21 @@ static_assert(1 >= 0
 #endif
 
 /**
- * Test Extruder Pins
+ * Test Extruder Stepper Pins
  */
-#if ENABLED (SWITCHING_EXTRUDER)
-  #if EXTRUDERS > 2
-    #if !PIN_EXISTS(E1_STEP) || !PIN_EXISTS(E1_DIR) || !PIN_EXISTS(E1_ENABLE)
-      #error "E1_STEP_PIN, E1_DIR_PIN, or E1_ENABLE_PIN not defined for this board."
-    #endif
-  #endif
-#elif EXTRUDERS > 4
+#if E_STEPPERS > 4
```

Comment:
As long as this test uses `E_STEPPERS` then all typed of extruders will be handled.